systemtap/lilac.yaml

```diff
@@ -1,14 +1,10 @@
 # This file is auto-generated by n2y
 
 maintainers:
-  - github: lijinpei
-    email: leekingp1994@gmail.com
+  - github: ideal
+    email: idealities@gmail.com
 
-build_prefix: archlinuxcn-x86_64
-
-pre_build: aur_pre_build
-
-post_build: aur_post_build
+build_prefix: extra-x86_64
 
 update_on:
   - aur: systemtap
```

Comment:
This line should be changed. With the current configuration, the package will be rebuilt when the AUR package is updated. In this case the same PKGBUILD is processed again and thus wasting server resources.

Reply:
```
diff --git a/systemtap/lilac.yaml b/systemtap/lilac.yaml
index d90102f31..b0f5b6fd1 100644
--- a/systemtap/lilac.yaml
+++ b/systemtap/lilac.yaml
@@ -7,4 +7,4 @@ maintainers:
 build_prefix: extra-x86_64

 update_on:
-  - aur: systemtap
+  - aur
```

Will this be OK?

Reply:
Well,  `- aur` is equivalent to `- aur: systemtap` - lilac determines the AUR package name if it's omitted. You may want to use `manual` in this case.